perception/image_projection_based_fusion/launch/roi_cluster_fusion.launch.xml

```diff
@@ -72,6 +73,7 @@
       <param name="use_iou_y" value="false"/>
       <param name="iou_threshold" value="0.35"/>
       <param name="rois_number" value="$(var input/rois_number)"/>
+      <param from="$(var param_path)"/>
```

Comment:
```suggestion
      <param from="$(var sync_param_path)"/>
```

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/2175/commits/7807a448f3c772d49ebe50bc6f5fe3863ead6a64